modules/terraform/show.go

```diff
@@ -0,0 +1,32 @@
+package terraform
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type Format int
+
+const (
+	JSON Format = iota
+	Human
+)
```

Comment:
Nice use of enum! 👍

Reply:
NIT: This should probably be called `PlanJSON` or `PlanHuman`, in case we have other `JSON` in the future related to terraform (since the namespace is shared with the whole package).